openassessment/assessment/api/staff.py

```diff
@@ -54,18 +54,27 @@ def assessment_is_finished(submission_uuid, requirements):
     Returns:
         True if a staff assessment has been completed for this submission or if not required.
     """
-    if requirements and requirements.get('staff', {}).get('required', False):
```

Comment:
Yeah, they're unpacked by step prior to hitting this code, I had previously been double-unpacking. (See https://github.com/edx/edx-ora2/blob/efischer/enable_staff_grading/openassessment/workflow/models.py#L252)

Good idea about renaming, that ought to make things clearer.